os/include/net/lwip/stats.h

```diff
@@ -481,14 +481,14 @@ void stats_init(void);
 
 /* Display of statistics */
 #if LWIP_STATS_DISPLAY
-void stats_display(void);
+int stats_display(int argc, char **argv);
```

Comment:
using argc and argv at API except main function is not proper. please change definition.

Reply:
Currently, netcmd calls stats_display function directly in the main. Because this function is not highly related with netmon, I will remove all chagnes.